Packs/RecordedFuture/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,80 @@
+#### Pack
+
+##### Recorded Future Integration
+
+- Note: pack was renamed from "RecordedFuture v2" to "Recorded Future Integration".
```

Comment:
I think it should be:
```suggestion
- Note: pack was renamed from "RecordedFuture v2" to "Recorded Future Recorded Future Intelligence".
```

Reply:
Changed to: 
`Note: pack was renamed from "RecordedFuture v2" to "Recorded Future Intelligence".`